source/includes/reference/_transfers.md

```diff
@@ -47,7 +47,7 @@ curl -X POST https://api.sandbox.transferwise.tech/v1/transfers \
 }
 ```
 
-Transfer is a payout order to recipient account based on a quote. Once created then a transfer needs to be funded during next 5 working days. 
+Transfer is a payout order to recipient account based on a quote. Once created then a transfer needs to be funded during next 10 working days(based on the source currency). 
```

Comment:
I guess such a PR demands nit-picking :P

I would say "*A transfer* is a payout order to *a* recipient account based in a quote."

The second sentence is also slightly awkward, perhaps "Once created, a transfer...".

Also there is always space surrounding parentheses in English. "days (based ... "

Reply:
I should say, space between unenclosed words and parens, not trailing punctuation.